pkg/talos/talos_cluster_health_data_source.go

```diff
@@ -127,7 +190,7 @@ func (d *talosClusterHealthDataSource) Read(ctx context.Context, req datasource.
 		return
 	}
 
-	readTimeout, diags := state.Timeouts.Read(ctx, 10*time.Minute)
+	readTimeout, diags := state.Timeouts.Read(ctx, 1*time.Minute)
```

Comment:
is that a default timeout? 1 minute might be too small?

Reply:
ah I forgot to update after testing